Georeferencing-Survey-12d-3/readme.md

```diff
@@ -0,0 +1,64 @@
+
+### Intent
+
+Survey data is collected from the very beginning of a project and the position and elevation of any other data in the project must be tied back to accurate survey.  
+Survey data starts with points and strings which are an ordered set of points (vertices) vertices connected by segments.
+So a survey point is just a survey string with only one vertex.
+ 
+As well as collecting the (x,y) and sometimes a z for each vertex, surveyors also collect properties for the entire string, properties for each vertex of the string and properties for each segment of the string.
+For example when picking up the top of a buried pipe, the Quality level can vary for each vertex and for each segment of the pipe.
+
+However I have no idea how these prperties can be represented in IFC.
+So as a start I have mapped the Property set for the vertices and segments to Property Sets for IfcAnnotation with:
+
+- Property sets with properties for the entire string
+- Property set Vertex with the properties for each vertex with the property name for a vertex appended by the number for the vertex (starting at 1)
+- Property set Segment with the properties for each segment with the property name for a segment appended by the number for the segment (starting at 1)
```

Comment:
Proposal:

I'd introduce an `IfcAnnotation` for each individual point, assign the corresponding IfcCartesianPoint as its geometric representation, and assign the corresponding PropSet for each individually.

You can also make use of the `IfcRoot.Name` attribute (part of the `IfcAnnotation`) for naming of points / string lines, etc.

*NOTE* I'd except this text to be part of the pull request in order to discuss in the pull request, not as part of the "finalized" Readme :)

Reply:
What about the geometry of the segment between each vertex ?
And the Property Sets that go with each segment ?

Reply:
That would be another IfcAnnotation with its own geometry (line?) and properties.

Reply:
A new unit test Georeferencing-Survey-12d-4 as been added with such an alternate structure for the survey string line with string line property sets, vertex property sets and segment property sets.